### ISSUE TITLE
Configure HTML log generation by WATO

### ISSUE THREAD

**Issue by simonmeggle** [simonmeggle/robotmk#56]
It should be possible to configure in WATO that robot also generates the HTML report.  
A _historical_ recording of logs is important to investigate the reason for unknown errors.

**Comment by simonmeggle** [comment]
log/report should be configurable by suite (only setting "log"). WATO should provide the following settings: 

* None (do not log at all) => ''
* last (create `$SUITENAME`-report.html)
* number (create `$SUITENAME`-report-`$TIMESTAMP`.html) 

```
cache_time: 3600
console: ''
execution_type: external_spooldir
log: ''
report: ''
robotdir: C:\ProgramData\checkmk\agent\robot
suites:
  e2e-foo:
    log: 100    # keep the last 100 HTML reports & output.xml
    variable:
      myvar1: foo
      myvar2: bar
```

**Comment by simonmeggle** [comment]
closed similar #42 for this one

**Comment by simonmeggle** [comment]
Done
